apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx

```diff
@@ -220,13 +221,43 @@ const NftViewer = ({ route }: Props) => {
     return null;
   }, [isLoading, nftMetadata]);
 
-  const nftImage = (
-    <NftImage
-      resizeMode="contain"
-      style={styles.image}
-      src={nftMetadata?.media}
-      status={nftStatus}
-    />
+  const mediaType = useMemo(() => getMetadataMediaType(nftMetadata, "big"), [
+    nftMetadata,
+  ]);
+  const MaybePressableNftImageViewer = useCallback(
```

Comment:
not a good practice to create component on the fly inside the render. it will force React to re-mount the whole sub tree.
prefer to inline this code in the returned.

Reply:
Changed for an inline solution instead 👌